frontend/client/styles/components/footer.scss

```diff
@@ -2,34 +2,61 @@
 
 #footer {
   display: flex;
+  // `margin-top: auto;` pins the footer to the bottom of the app, see https://stackoverflow.com/a/33924704
+  margin-top: auto;
 }
 
 .footer-not-login {
-  padding-right: 101px;
+  @include media('>phone') {
+    padding-right: 101px;
+  }
 }
 
 .footerContainer {
-  position: relative;
   display: flex;
-  width: 100%;
-  justify-content: space-between;
   align-items: center;
-  margin-left: 98px;
-  margin-right: 98px;
-
-
+  justify-content: space-between;
+  width: 100%;
+  @include media('>phone') {
+    flex-direction: row;
+    justify-content: space-between !important;
```

Comment:
Is there any reason why this needs to be important tagged and included here? Testing in chrome console showed no difference when removed.